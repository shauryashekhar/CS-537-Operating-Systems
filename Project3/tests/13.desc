Checking JID count, white space and no exit in batchfile 