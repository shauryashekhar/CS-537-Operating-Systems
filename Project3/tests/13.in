echo "0"
sleep 2 &
echo "2"



jobs
jobs
