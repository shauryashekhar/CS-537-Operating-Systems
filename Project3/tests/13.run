./mysh tests/13.in
