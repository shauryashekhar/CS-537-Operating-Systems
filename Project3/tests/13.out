echo "0"
"0"
sleep 2 &
echo "2"
"2"



jobs
1 : sleep 2
jobs
1 : sleep 2
